sdks/java/core/src/main/java/org/apache/beam/sdk/runners/inprocess/InProcessPipelineRunner.java

```diff
@@ -154,6 +154,18 @@
      * timers that fired to produce this bundle.
      */
     Instant getSynchronizedProcessingOutputWatermark();
+
+    /**
+     * Return a new {@link CommittedBundle} that is like this one, except calls to
+     * {@link #getElements()} will return the provided elements. This bundle is unchanged.
+     *
+     * <p>
+     * The value of {@link #getSynchronizedProcessingOutputWatermark()} of the returned
```

Comment:
`The {@link #getSynchronizedProcessingOutputWatermark() synchronized processing watermark} of the returned...`